includes/transformer/class-post.php

```diff
@@ -280,6 +280,9 @@ protected function get_attachment() {
 			return array();
 		}
 
+		// phpcs:ignore Universal.Operators.DisallowShortTernary
+		$max_media = \get_post_meta( $this->item->ID, 'activitypub_max_image_attachments', true ) ?: \get_option( 'activitypub_max_image_attachments', ACTIVITYPUB_MAX_IMAGE_ATTACHMENTS );
```

Comment:
this looks a bit ugly... can we maybe add a function for that?

Reply:
```suggestion
		$max_media = \get_post_meta( $this->item->ID, 'activitypub_max_image_attachments', true );
		if ( ! is_numeric( $max_media ) ) {
			$max_media = \get_option( 'activitypub_max_image_attachments', ACTIVITYPUB_MAX_IMAGE_ATTACHMENTS );
		}
```
Would this be better?